charts/brig/templates/deployment.yaml

```diff
@@ -45,7 +45,7 @@ spec:
           command:
           - sh
           - -c
-          - echo turn:127.0.0.1:3478 >> /etc/wire/brig/turn/turn-servers.txt
+          - echo -e "{{- range .Values.turnStatic.v1 }}{{ . }}\n{{- end }}" | sed '$d' > /etc/wire/brig/turn/turn-servers.txt
```

Comment:
instead of an empty dir "turn-servers" and an echo init container, wouldn't it be cleaner to create a second configmap and mount that?
(adapt the TODO comment above, and ensure to add another checksum/turn-config annotation)

Reply:
Yes, that's much better, will do so.